packages/babel-preset-shopify/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  presets: [
+    require('babel-preset-es2015'),
+    require('babel-preset-react'),
+    require('babel-preset-stage-2')
+  ],
+  plugins: [
+    require('babel-plugin-transform-class-properties'),
+    require('babel-plugin-transform-export-extensions')
+  ]
```

Comment:
If this is going to get listed it's probably going to fail, since it's missing the trailing commas.